tox.ini

```diff
@@ -1,29 +1,40 @@
 [tox]
-skipsdist = True
-envlist = py39, py310, black, flake8, docs
+envlist = py38, py39, py310, black, flake8, docs
```

Comment:
This is mainly useful for developers who want to run `tox` without arguments and have it cover a reasonable subset of possible environments.  For that purpose it may not be necessary to test more than one Python version, but either way is good by me.